# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-auxil/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-auxil/.cargo-checksum.json b/third_party/rust/gfx-auxil/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/gfx-auxil/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"617d82a8d7a530be460d69af00830a3e1e4b807ff173d262910bec3947a561b6","src/lib.rs":"f64e02aec86b3c391e85c41ea4f4cb9eb0c5868f67e5461261e15e866df5ee4d"},"package":"1694991b11d642680e82075a75c7c2bd75556b805efa7660b705689f05b1ab1c"}
\ No newline at end of file